doc.ddoc

```diff
@@ -67,6 +19,7 @@ $(SCRIPTLOAD js/cssmenu.js)
 </html>
 _=
 
+ROOT = .
```

Comment:
I think

    ROOT = .
    ROOT = ..
    $(ROOT)/foo

are clearer and easier to get right than

    ROOT =
    ROOT = ../
    $(ROOT)foo

But if there's an upside to doing it the other way, I can do that of course.